commons/java/openjdk11/Dockerfile

```diff
@@ -2,5 +2,7 @@ FROM cloudsuitetest/debian:base-os
 
 RUN apt-get update -y && apt-get install -y --no-install-recommends openjdk-11-jdk-headless \
         && rm -rf /var/lib/apt/lists/*
-# TODO: Fix this link for other architectures.
-ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64/
+
+ARG EXTERNAL_ARG
+ENV JAVA_HOME=${EXTERNAL_ARG}
```

Comment:
I don't understand why you are making is difficult and try to pass this as a parameter from outside.
As suggested before this can be done easily by checking the arch inside the container (e.g., using `uname -m`) and then based on that setting the path. There is no need to pass anything to this from outside.

Reply:
You can see how that simple command gives you the arch and then based on that you can decide whatever you want to do:

```
➜  ~ docker run -it cloudsuitetest/debian:base-os@sha256:167f9388f0f7ed357d6528cf0fcbd2d9b82be33ede0951d150768b1f087eaf94 uname -m
WARNING: The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64) and no specific platform was requested
aarch64
➜  ~ docker run -it cloudsuitetest/debian:base-os@sha256:a32e9bc5daa762d18301e87f0669fbf6c5d5ecd5407ed432812f5c619c8d618c uname -m
WARNING: The requested image's platform (linux/riscv64) does not match the detected host platform (linux/amd64) and no specific platform was requested
riscv64
➜  ~ docker run -it cloudsuitetest/debian:base-os@sha256:530e3b320138f2bb8f1bd4e24d54d02eea5c3128531602cb5acae408e0394d8d uname -m
x86_64
```

Reply:
I've tried different approaches. Even the [stackoverflow link](https://stackoverflow.com/questions/34911622/dockerfile-set-env-to-result-of-command) that you have sent in a previous PR. The problem is that we just can use ENV command and it only accepts a constant input. I found no way to dynamically set it except through this argument passing.